src/hooks/api-queries.ts

```diff
@@ -118,3 +126,22 @@ export const useListFilesQuery = () =>
     queryKey: ["files"],
     queryFn: () => filesApi.listFiles().catch(handleErrorWithMessage("Error listing files")),
   });
+
+export const useListProjectMilestonesQuery = (params: ListProjectMilestonesRequest) => {
```

Comment:
Hmm, maybe we should consider adding docs to those exported consts as well? What do you think?

Reply:
I agree.